### PR TITLE
Add example for update command

### LINKED ISSUE #2518: Better docs for `update`
**Is your feature request related to a problem? Please describe.**
I was having a lot of trouble figuring out how to table a table, update 1 column, and then output a new table with that column updated. The `update` command's documentation only shows how to add a fixed value to the table, and not perform transformations of the old value.

**Describe the solution you'd like**

Better documentation, and maybe a reference in the cookbook.

**Describe alternatives you've considered**
I'm fairly new to query based languages like Nu so I'm not sure.

**Additional context**

For anyone reading this and wondering how to use `update`:

Say you have a table like this:
```
❯ open fonts.json | first 3
───┬───────────┬───────┬────────────────
 # │   Name    │ Value │      Tags
───┼───────────┼───────┼────────────────
 0 │ Legacy    │     0 │ [table 1 rows]
 1 │ Arial     │     1 │ [table 2 rows]
 2 │ ArialBold │     2 │ [table 0 rows]
───┴───────────┴───────┴────────────────
```

and you want to perform a transformation on `Tags` to make it readable. You use the `update` command like this:
```
❯ open fonts.json | update Tags { get Tags | str collect ", " } | first 3
───┬───────────┬───────┬──────
 # │   Name    │ Value │ Tags
───┼───────────┼───────┼──────
 0 │ Legacy    │     0 │ Deprecated
 1 │ Arial     │     1 │ Foo, Bar
 2 │ ArialBold │     2 │
───┴───────────┴───────┴──────
```

### PR DESCRIPTION
**Pull request by gillespiecd**
Closes #2518 

1. Add more docs and an example for `update`
1. Add test cases for `rename` examples

```
Examples:
  Update a column value
  > echo "{a: 1, b: 2}" | from json | update a 'nu
```

### CODE REVIEW

File: crates/nu-cli/src/examples.rs [hunk 1]
```diff
@@ -20,6 +20,7 @@ pub fn test(cmd: impl WholeStreamCommand + 'static) {
         whole_stream_command(Each {}),
         whole_stream_command(cmd),
         whole_stream_command(StrCollect),
+        whole_stream_command(FromJSON),
```
Comment (andrasio) [inline crates/nu-cli/src/examples.rs:23]: We try to limit the commands here. Can you please remove this and use table literals in the examples instead? Thank you!

File: crates/nu-cli/src/commands/update.rs [hunk 2]
```diff
@@ -49,6 +50,18 @@ impl WholeStreamCommand for Update {
     ) -> Result<OutputStream, ShellError> {
         update(args, registry).await
     }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "Update a column value",
+            example: r#"echo "{a: 1, b: 2}" | from json | update a 'nu'"#,
```
Comment (andrasio) [inline crates/nu-cli/src/commands/update.rs:57]: Here for instance, please try `echo [[a, b]; [1, 2]]` table literal instead.

**Comment by gillespiecd** [comment]
@andrasio Updated to use table literals, looks cleaner 👍

**Comment by andrasio** [comment]
> @andrasio Updated to use table literals, looks cleaner 👍

Indeed. Table literals were requested for quite some time. They are here, finally!

**Comment by andrasio** [comment]
Thanks again!

**Pull request merged**